### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by janvanmansum**
@jo-pol @ekoi @PaulBoon @vesaakerman please have a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,24 @@ is not present in the Fedora repository or `stage-digital-object-set` cannot be
 the program terminates with an error.
 
 For each digital object the last version of each (managed) datastream, a `fo.xml` and `cfg.json` file are downloaded.
-The `fo.xml` file includes the inline datastreams DC, EMD, AMD, PRSQL, DMD.
-- the datastream AUDIT is skipped completely
-- RELS-EXT is exported into to the "relations"-map in the file `cfg.json` ([Digital Object Configuration])
+The `fo.xml` file includes the inline datastreams `DC`, `EMD`, `AMD`, `PRSQL`, `DMD` as far as they are present. As
+for the other datastreams:
+
```
Comment (janvanmansum) [inline README.md:25]: Use the asterisks for bulleted lists. This seems to be the standard way in markdown.

File: README.md [hunk 2]
```diff
@@ -19,12 +19,24 @@ is not present in the Fedora repository or `stage-digital-object-set` cannot be
...
+* The datastream `AUDIT` is skipped completely.
+* `RELS-EXT` is exported into to the "relations"-map in the file `cfg.json` ([Digital Object Configuration])
   Fedora PIDs that reference digital objects in the same dataset are replaced by the appropriate SDO-name.
-- EASY-FILE-METADATA is downloaded separately as are eventual other inline datastreams not mentioned above
-- checksums and dataset related PIDs in the `fo.xml` are skipped
+* Other datastreams, such as `EASY_FILE_METADATA`,  are downloaded separately regardless whether they are inline or
+  managed.
+
+Checksums and dataset related PIDs in the `fo.xml` are skipped.
+
```
Comment (janvanmansum) [inline README.md:34]: @jo-pol Some things that are not clear yet:

File: src/main/scala/nl/knaw/dans/easy/export/FOXML.scala [hunk 3]
```diff
@@ -32,7 +32,7 @@ object FOXML {
     *   these IDs should be identical between EASY-fedora instances
     *   unless a release that changed audiences was not applied everywhere
     */
-  val plainCopy = Seq("DC", "EMD", "AMD", "PRSQL", "DMD")
+  val downloadInFoxml = Seq("DC", "EMD", "AMD", "PRSQL", "DMD")
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/export/FOXML.scala:35]: `plainCopy` does not really describe what this does.

Comment (jo-pol) [inline src/main/scala/nl/knaw/dans/easy/export/FOXML.scala:35]: Thanks, I was struggling with that one.

File: README.md [hunk 4]
```diff
@@ -19,12 +19,24 @@ is not present in the Fedora repository or `stage-digital-object-set` cannot be
...
+**TODO (add to documentation)**:  
+    
+* **WHAT "dataset related PIDs" ARE REFERRED TO HERE?**
+* **What happens to the checksum of the inline datastreams exported in the foxml? (Exported along with the datastream?)**
+* **What happens to the checksum of datastreams downloaded separately? (For the sentence about the checksums above I
+  gather that these checksums are not downloaded. Are the checksum settings downloaded? I.e. will the checksum be
+  created when ingesting the staged dataset?)**
```
Comment (jo-pol) [inline README.md]: To the point question: I remove `<foxml:contentDigest>` and after ingest I don't see it automagically regenerated!

**Pull request merged**